include/spdlog/tweakme.h

```diff
@@ -93,7 +93,7 @@
 // In this case spdlog will try to include <fmt/format.h> so set your -I flag
 // accordingly.
 //
-// #define SPDLOG_FMT_EXTERNAL
+#define SPDLOG_FMT_EXTERNAL
```

Comment:
There is no need to touch this file, you can add `SPDLOG_FMT_EXTERNAL=1` in CMakeLists.txt:
* https://github.com/hunter-packages/spdlog/blob/v0.16.3-p1/CMakeLists.txt#L48-L52

Reply:
Few other things missing as well, I don't see `Threads::Threads` and `android_log::android_log` targets for example. Have you used patches from `hunter-0.16.3` branch? https://github.com/hunter-packages/spdlog/commits/hunter-0.16.3

Reply:
They are used in test, bench and example so I don't think it's needed to include them for the project.

Reply:
> They are used in test, bench and example so I don't think it's needed to include them for the project.

I see it's linked to the main `spdlog` target:
* https://github.com/hunter-packages/spdlog/blob/hunter-0.16.3/CMakeLists.txt#L54-L56
* https://github.com/hunter-packages/spdlog/blob/hunter-0.16.3/CMakeLists.txt#L68-L71

Reply:
For 1.2.1, Threads used in tests.
https://github.com/hunter-packages/spdlog/blob/hunter-1.2.1/tests/CMakeLists.txt#L3
also in bench https://github.com/hunter-packages/spdlog/blob/hunter-1.2.1/bench/CMakeLists.txt#L32
and example https://github.com/hunter-packages/spdlog/blob/hunter-1.2.1/example/CMakeLists.txt#L32

There isn't any `find_package` in the main `CMakeLists.txt` any more.

Reply:
> There isn't any find_package in the main CMakeLists.txt any more.

I haven't run any tests but I think that simply is an upstream issue. Threads used by main library, e.g. here:
* https://github.com/hunter-packages/spdlog/blob/hunter-1.2.1/include/spdlog/details/os.h#L334

Meaning every library that use spdlog headers should use Threads::Threads while linking. You have no problems at this point because it's a header-only library. You will experience issues when using real code.

Anyway I'm ready to do the merge if this version is working with Hunter CI. If any problem will occur in future then Hunter testing example should be improved to cover this case.